test/infrastructure/docker/e2e/local-e2e.conf

```diff
@@ -10,6 +10,12 @@ images:
   loadBehavior: tryLoad
 - name: gcr.io/k8s-staging-cluster-api/kubeadm-control-plane-controller-amd64:dev
   loadBehavior: tryLoad
+- name: quay.io/jetstack/cert-manager-webhook:v0.11.1
```

Comment:
quay was having a real hard time and kept 500ing so this should reduce the issue where a pod can't pull from quay when running locally. Note: This does not affect CI at all.